geopandas/tests/test_geoseries.py

```diff
@@ -84,7 +80,7 @@ def test_align(self):
         assert a1["B"].equals(a2["B"])
         assert a1["C"] is None
 
-    @pytest.mark.skipif(not HAS_PYPROJ, reason="pyproj not available")
+    @pytest.mark.skipif(not compat.HAS_PYPROJ, reason="pyproj not available")
```

Comment:
I understand this is just some form of linting but if you can limit the changes in the PR to relevant lines only, it would help the review.